mdgen/docker/integration_tests/features/feature/launch.feature

```diff
@@ -0,0 +1,33 @@
+Feature: The java app
+
+  Background:
+    Given the java app exists
+    And the login credentials exist
+
+
+  Scenario: Launching the java app
+    When I run the java executable with no parameters
+    Then I see that the application complains about missing parameters
+
+  Scenario: Launching the java app with decent parameters
```

Comment:
I think that this test and the one below might be functionally equivalent? They both generate for the connector node using RSA, and they both supply the certificates, rather than the saml signing cert being generated? This tests the new use case that we've added but I din't think there's a test for the original scenario?